_posts/2019-01-18-future-typescript-eslint.md

```diff
@@ -0,0 +1,43 @@
+>
+>In a survey we ran in VS Code a few months back, the most frequent theme we heard from users was that the linting experience left much to be desired. Since part of our team is dedicated to editing experiences in JavaScript, our editor team set out to add support for both TSLint and ESLint. However, we noticed that there were a few architectural issues with the way TSLint rules operate that impacted performance. Fixing TSLint to operate more efficiently would require a different API which would break existing rules (unless an interop API was built like what wotan provides).
+>
+>Meanwhile, ESLint already has the more-performant architecture we're looking for from a linter. Additionally, different communities of users often have lint rules (e.g. rules for React Hooks or Vue) that are built for ESLint, but not TSLint.
+>
+>Given this, our editor team will be focusing on leveraging ESLint rather than duplicating work. For scenarios that ESLint currently doesn't cover (e.g. semantic linting or program-wide linting), we'll be working on sending contributions to bring ESLint's TypeScript support to parity with TSLint. As an initial testbed of how this works in practice, we'll be switching the TypeScript repository over to using ESLint, and sending any new rules upstream.
+
+We, just like a lot of the JavaScript and TypeScript communities, were very excited about this announcement. Several members of the ESLint team have been working on improving TypeScript compatibility of the past couple of years, including James Henry, Kai Cataldo, Kevin Partington, Toru Nagashima, and Nicholas Zakas. That work focused mainly on the TypeScript parser, [`typescript-eslint-parser`](https://github.com/eslint/typescript-eslint-parser) (and partly on [`eslint-plugin-typescript`](https://github.com/bradzacher/eslint-plugin-typescript), which was not maintained by the ESLint team but had been maintained by Nicholas and James until recently). The Typescript parser would undoubtedly become the centerpiece of the TypeScript-in-ESLint story going forward; as such, we wanted to make sure that it would be properly maintained.
+
+## Announcing the `typescript-eslint` project
+
+James Henry, who has long been the driving force behind TypeScript compatibility for ESLint, has started the [`typescript-eslint`](https://github.com/typescript-eslint/typescript-eslint) project as a centralized repository for all things related to TypeScript ESLint compatibility. This will be the new home of the TypeScript parser, `eslint-plugin-typescript`, and any other utilities that will make the TypeScript ESLint experience as seamless as possible.
+
+While the ESLint team won't be formally involved in the new project, we are fully supportive of James' efforts and will be keeping lines of communication open to ensure the best ESLint experience for TypeScript developers.
+
+## What this means going forward
+
+All of this means a few things that are important to understand:
```

Comment:
Tone-wise, “important to understand” could be read as being a little pedantic (even though I know that’s not the intention).

I think that this section doesn’t actually need an intro sentence. Just having the header and list seems like it gets the point across just fine.

Reply:
Sounds good to me.